### PR TITLE
Remove <br> for layout; add home stylesheet

### PR DESCRIPTION
**Pull request by openjck**

### CODE REVIEW

File: dashboard/static/css/home.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+form#add-push-application input[type="submit"] {
+    display: block;
+}
```
Comment (groovecoder) [inline dashboard/static/css/home.css:3]: How would you feel about leaving all styles in a single stylesheet for the whole site?

Comment (openjck) [inline dashboard/static/css/home.css:3]: We could! That would make some things easier in the short term. At the same time, it doesn't cost much to have a separate stylesheet for the homepage, and we did spend a lot of time moving stuff out of that big monolithic _mdn-screen.css_ file on MDN a while back.

Comment (groovecoder) [inline dashboard/static/css/home.css:3]: What if we have `main.css` and `home.css` source files to keep the code separate and organized, but then have it all compile into a single `styles.css` that's included on all pages?

At some point the bandwidth saved by downloading just the optimized CSS a page needs is overcome by the requests and bandwidth spent downloading a different (though optimized) CSS resource on so many pages.

Comment (openjck) [inline dashboard/static/css/home.css:3]: Yeah, I'm good with that for now. I'll update it.

**Comment by openjck** [comment]
Updated

**Pull request merged**